_lib/wordpress_sub-page_processor.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import json
+import os.path
+import requests
+
+
+def posts_at_url(url):
```

Comment:
Is this function exactly the same in all of our wordpress processors?  If so, what are your thoughts on throwing it into a helpers.py and importing it each time?